src/components/sandbox/contract-call.tsx

```diff
@@ -111,35 +125,54 @@ export const ContractCall = (props: any) => {
           {styles => (
             <Box width="100%" position="absolute" style={styles}>
               <Formik enableReinitialize initialValues={initialValues} onSubmit={onSubmit}>
-                {({ handleSubmit }) => (
-                  <Box
-                    as="form"
-                    // @ts-ignore
-                    onSubmit={handleSubmit}
-                    method="post"
-                    style={{ width: '100%' }}
-                  >
-                    <Flex width="100%">
-                      <Stack spacing="base" width="100%">
-                        <Field
-                          label="Contract address"
-                          placeholder="STRYYQQ9M8KAF4NS7WNZQYY59X93XEKR31JP64CP"
-                          name="contractAddress"
-                        />
-                        <Field
-                          label="Contract Name"
-                          placeholder="hello-world-contract"
-                          name="contractName"
-                        />
-                        <Box>
-                          <Button ref={buttonRef as any} type="submit" isLoading={isLoading}>
-                            Fetch
-                          </Button>
-                        </Box>
-                      </Stack>
-                    </Flex>
-                  </Box>
-                )}
+                {({ handleSubmit, setValues }) => {
```

Comment:
forkik exposes hooks too, to avoid nesting in the jsx, i saw recently

Reply:
I was reading and using the `useFormik` hook but my understanding was that the render props method is the main way, and they don't recommend using the hook as it's more for internal usage only. I do prefer the hook.